example-grafana-dashboard-overview.json

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
What's the license for the JSON definitions? Should we also add a license header as a comment?

Reply:
AFAIR JSON does not support comments. People sometimes use [workarounds](https://www.freecodecamp.org/news/json-comment-example-how-to-comment-in-json-files/) but that adds extra data and might confuse or fail validation (if present). The [JSON model itself](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/view-dashboard-json-model/) also does not contain a field to specify license information.

In the long run, Dashboards should probably be hosted [here](https://grafana.com/grafana/dashboards/) instead of inside the repository, so that people can find/add them easily to their Grafana instance. OTOH keeping a versioned dashboard along with the code also seems to be a good idea.

I have no idea how to solve this problem, except extending the README to state that the example Dashboards are also covered by the BSD-2-clause license. What do you think @mmuehlenhoff?

Reply:
Fixed :-)